addons/website_payment/static/src/js/website_payment_form.js

```diff
@@ -7,6 +7,9 @@ odoo.define('website_payment.website_payment_form', require => {
     const _t = core._t;
 
     const websiteSalePaymentMixin = {
+        events: _.extend({}, checkoutForm.prototype.events || {}, {
+            'change .fee_impact': '_onFeeParameterChange',
```

Comment:
Is this one of our class?

Reply:
`fee_impact` is a class I added in the template to identify the elements that need to be taken into account for the fee recomputation. Maybe the name should be prefixed somehow (`o_fee_impact`) ?

Reply:
Or even `o_wpayment_fee_impact` to indicate it comes from the website_payment module. Whatever's fine but always at least prefix by o_ to differentiate our classes.